acceptance/suites/tests/service_framework/reload_updated_crl.rb

```diff
@@ -1,4 +1,4 @@
-test_name '(SERVER-1380) Server loads updated CRL after being reloaded'
+test_name 'Server reloads updated CRL without a restart'
```

Comment:
The test was originally exercising a service reload to see that the new CRL was being used.  With the addition of the file system watcher and associated Jetty reload, a service HUP should no longer be necessary.  I thought it might be best to just change the test to validate that the runtime reload occurs rather than doing the HUP, although maybe an argument could be made for testing both the HUP and filesystem watcher.